examples/http/index.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2016-2017, Pulumi Corporation.  All rights reserved.
+
+import * as azure from "@pulumi/azure";
+import * as eventhub from "@pulumi/azure/eventhub";
```

Comment:
Why are you importing `eventhub`? Maybe I'm not seeing it.

Reply:
Fixed!